packages/scheduler/src/forks/Scheduler.js

```diff
@@ -427,9 +429,8 @@ let taskTimeoutID = -1;
 // thread, like user events. By default, it yields multiple times per frame.
 // It does not attempt to align with frame boundaries, since most tasks don't
 // need to be frame aligned; for those that do, use requestAnimationFrame.
-// TODO: Make these configurable
-let frameInterval = 5;
-const continuousInputInterval = 50;
+let frameInterval = frameYieldMs;
```

Comment:
`maxInterval` needs to be configurable, too. That's what will control how long, for example, network requests are blocked for.